general/table.hpp

```diff
@@ -208,6 +208,7 @@ void Transpose (const Table &A, Table &At, int ncols_A_ = -1);
 Table * Transpose (const Table &A);
 
 ///  Transpose an Array<int>
+/** @note The column (TYPE II) indices in each row of @a At will be sorted. */
 void Transpose(const Array<int> &A, Table &At, int ncols_A_ = -1);
```

Comment:
Maybe this comment could be expanded to explain how an `Array<int>` is used to represent a table? The way I understand it is that the array represents a table where each row $i$ has exactly one marked column, so there exists the connection $(i, A[i])$, but this wasn't so obvious to me.

Reply:
Done in b080c556a365f19d52733cad1d05f00ccc54efc1